Sources/TuistGenerator/Generator/ProjectFileElements.swift

```diff
@@ -128,6 +128,10 @@ class ProjectFileElements {
             files.insert(entitlements)
         }
 
+        if let modulemap = target.modulemap {
```

Comment:
👍 - awesome this means `modulemaps` will automatically appear in Xcode navigator - no longer need to manually add them to `additionalFiles` in the manifest! 🎉